Node.cpp

```diff
@@ -1,9 +1,33 @@
 #include "Node.h"
+#include <limits>
 
-Node::Node(void) {
-    exists = true;
-}
+using namespace std;
+
+Node::Node(void):
+	pos(NodeCoordinate(0,0)),
+	exists(false)
+{}
+
+Node::Node(NodeCoordinate pos):
+	pos(pos),
+	exists(true)
+{}
 
 Node::operator bool() const {
     return exists;
 }
+
+void Node::resetPathData()
+{
+	gScore = numeric_limits<unsigned>::max();
+	fScore = numeric_limits<unsigned>::max();
+	previous = nullptr;
+	evaluated = false;
+	direction = NONE;
+}
+
+void Node::setScores(unsigned gScore, unsigned fScore)
```

Comment:
No not really. I thought I would have needed it more.
Removed in 57795a47a70c9b97f6c574036e736d6942714bef